src/versionbits.cpp

```diff
@@ -105,6 +104,42 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex*
     return state;
 }
 
+// return the numerical statistics of blocks signalling the specified BIP9 condition in this current period
+BIP9Stats AbstractThresholdConditionChecker::GetStateStatisticsFor(const CBlockIndex* pindexPrev, const Consensus::Params& params) const
+{
+    BIP9Stats obj;
+    
+    int elapsedBlocks;
+    int nPeriod = Period(params);
+    int nThreshold = Threshold(params);
```

Comment:
Actually just took out declarations for all those variables, assigned to return object directly and refer to those assignments downstream. (see https://github.com/bitcoin/bitcoin/pull/9571/commits/6eecfe0368525f70ff8b93914320356ae0070a0c)